docs/app/Examples/elements/List/Content/ListImageExample.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+import {List, ListItem} from 'stardust';
+
+export default class ListImageExample extends Component {
+  render() {
+    const helenAvatar = <img className='ui avatar image' src='http://lorempixel.com/100/100/people/1' />;
```

Comment:
Using the `<Image />` component makes them weirdly lopsided. I'm trying to figure it out but I thought I'd post it here if you have any ideas
<img width="318" alt="screen shot 2015-10-30 at 4 51 03 pm" src="https://cloud.githubusercontent.com/assets/6728034/10858914/810a676c-7f26-11e5-937e-06bebf426992.png">